vtex/faststore
# Feat: Next 13 - Global Section - Initial structure +`Alert` Section

## What's the purpose of this pull request?

This PR aims to add the initial structure from the global sections and migrate the Alert section from the pages router to the app router (next 13). 

Some components were moved from the `src` folder to the `app` folder to make it Server Components, with a few updates to make them use the updated path. 

<img width="845" alt="Screenshot 2024-06-10 at 19 26 49" src="https://github.com/vtex/faststore/assets/11325562/c242fdb4-b07f-4ab4-8b56-148e72208507">

## How to test it?

- Run the local server (`yarn dev` in core package);
- Make sure that all the current pages are working as expected (`/`, `/office` and some `/<slug>/p`);
- Make sure `/fs-next-update` is rendering with thew `Alert` section on the top of the page and without errors.

### Starters Deploy Preview

- https://github.com/vtex-sites/starter.store/pull/469
